pandas/tests/test_strings.py

```diff
@@ -377,11 +377,36 @@ def test_str_cat_align_mixed_inputs(self, join):
         with tm.assert_raises_regex(ValueError, rgx):
             s.str.cat([t, z], join=join)
 
-    def test_str_cat_raises(self):
+    @pytest.mark.parametrize('box', [Series, Index])
+    def test_str_cat_raises(self, box):
         # non-strings hiding behind object dtype
```

Comment:
need a test that validates that a bytes object is not wokring for all methods but decode